src/components/ui/TranslationLabel/TLabel.tsx

```diff
@@ -26,12 +26,18 @@ const dictionnariesMap = {
   language: translationLanguageDictionnaries,
 };
 
-type GlobalTranslationLabelType =
+export type GlobalTranslationLabelType =
   | TranslationLabelType
   | TranslationFeatureLabelType
   | TranslationLanguageLabelType
   | TranslationCategoryLabelType;
 
+export type GlobalTranslationDictionnaryType =
```

Comment:
```suggestion
type GlobalTranslationDictionnaryType =
```
**Note** : On accède tout de même aux infos quand on utilise TLabel, car c'est dans l'interface